generator.js

```diff
@@ -1,3 +1,15 @@
 module.exports = function(n) {
-  // do work here
+  let array = [];
+
+  for (i = 0; i < n; i++) {
+    array.push(i);
+  }
+
+  array.map(function(e) {
+    array.push(array[Math.floor(Math.random() * n * 10)]);
```

Comment:
Can you think of a more efficient way to handle trying to generate random numbers in the array. This is doubling the size of the array.